build.sbt

```diff
@@ -84,7 +85,7 @@ lazy val datagen = project
 
 lazy val examples = project
   .in(file("examples"))
-  .dependsOn(core, datagen)
+  .dependsOn(core, datagen, testkit)
```

Comment:
Using testkit instead of launching actual Kafka, useful for demonstration purposes